a/numbers.txt

```diff
@@ -6,7 +6,7 @@ I believe it's going to be similiar to:
     git checkout <remote_branch>
 
 1. I'm changing the first commit.
-2.
+2. Making a change to PR #2.
```

Comment:
Good new change is here. However,it's hard to see the conversation from the other commit.